_activities/gsoc.md

```diff
@@ -15,42 +15,58 @@ and analyze petabytes data from high-energy physics experiments, such as the Lar
 hosted at the CERN laboratory in Geneva, Switzerland. 
 Some of the questions that we collectively ask are: 
 
-- what are the fundamental blocks that make up our Universe? 
-- what is the nature of dark matter and dark energy?
-- what is the nature of the asymmetry between matter and antimatter? 
-- what was early Universe like? 
+- What are the fundamental blocks that make up our Universe?
```

Comment:
I disagree with this change - the sentence is a *list* of questions, so the start of each bullet should not be capitalised. (i.e., it's "I have various questions: who? what? where? and why?").